.github/workflows/pyopenms-wheels.yml

```diff
@@ -103,7 +103,7 @@ jobs:
           pip install -U pip
           pip install -U autowrap
           pip install -U pytest
-          pip install -U numpy
+          pip install numpy<2.0
```

Comment:
all of these `<` need to be inside quotes

Reply:
Ah, I see https://github.com/OpenMS/OpenMS/commit/7e751ffc51507a6e12b5eedccb05a1ff52e0561e